src/utils/ProfileUtils.ts

```diff
@@ -56,6 +58,22 @@ async function extractConfigsToImportedProfile(
     }
 }
 
+export async function installModsAndResolveConflicts(
```

Comment:
Food for a thought, not a change request:

Given that this is in the end a rather simple piece of code, it could be argued that having it in the utils is a bit of overkill, especially since we have another wrapper in the mixin that reduces duplication. Another thing that IMO speaks a bit against having it here is the fact that we need to pass the store as an argument. That's not inherently bad, but a weak signal that the split between UI component and helper library should be considered. For example, this helper is now not reusable if the call site doesn't have access to the store. Ideally we could import the store singleton to helpers but I _think_ that's currently unnecessarily difficult due to R2MM & TSMM having different store definitions.

I guess that my point is that I've no strong arguments one way or another, but my nitpick level opinion is that passing the store as an argument should be avoided if it's easy to do so.